pkgs/development/python-modules/patchpy/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  more-itertools,
+  pytestCheckHook,
+}:
+
+buildPythonPackage {
+  pname = "patchpy";
+  version = "2.0.3-unstable-2024-05-26";
```

Comment:
Any reason we're using an unstable version? PyPI seems to have versions listed, so the unstable suffix seems unnecessary:

https://pypi.org/project/patchpy/2.0.3/#history

Also, the latest version is now 2.0.4